ui/admin-portal/src/app/components/login/login.component.ts

```diff
@@ -77,13 +77,14 @@ export class LoginComponent implements OnInit {
     if (this.loading) { return; }
     this.loading = true;
 
-    const action = 'login';
-    this.reCaptchaV3Service.execute(action).subscribe(
-      (token) => this.loginWithToken(token),
-      (error) => {
-        console.log(error);
-      }
-    );
+    // const action = 'login';
+    // this.reCaptchaV3Service.execute(action).subscribe(
+    //   (token) => this.loginWithToken(token),
+    //   (error) => {
+    //     console.log(error);
+    //   }
+    // );
```

Comment:
So this will bypass the recaptcha token - is this what we want to do?

Reply:
Yes - recaptcha is for the chop. It is no longer needed since we moved to MFA. Checking is alreday disabled on the server - has been for a long time.
My current "In progress" issue #311 will pull it out altogether.